integration/combination/test_api_settings.py

```diff
@@ -0,0 +1,176 @@
+from io import BytesIO
+from pathlib import Path
```

Comment:
Not available in py2, use try/except like in https://github.com/aws/serverless-application-model/blob/develop/integration/helpers/base_test.py#L9

Reply:
will fix it